server/Windows/wf_wasapi.c

```diff
@@ -18,6 +18,7 @@
 //#define REFTIMES_PER_SEC  50000
 //#define REFTIMES_PER_MILLISEC  50
 
+#ifndef __MINGW32__
```

Comment:
Is there maybe a windows header we forgot to include so we can remove this define?

Reply:
These GUIDs are in mmdeviceapi.h which is already included. Not sure why these were added.

Reply:
might be something like https://stackoverflow.com/questions/31666420/undefined-reference-to-clsid-mmdeviceenumerator-and-iid-immdeviceenumerator